.gitignore

```diff
@@ -39,3 +39,6 @@ jspm_packages
 
 
 package-lock.json
+
+
+.idea
```

Comment:
developer IDE of choice stuff should not mess the repo. Please use global git ignore for that

Reply:
agree, gloabl git ignore config should be set.
Though we should also safeguard ourselves

Reply:
Thats why we have to have PR approvals :) Developers IDE of choice stuff have nothing to do in a repo. We cannot add all the different existing IDE settings in the gitignore. We just have to have an Eye open on the PRs :)